WordPress/UITestsFoundation/Screens/ActionSheetComponent.swift

```diff
@@ -1,7 +1,6 @@
-import UITestsFoundation
```

Comment:
This import is no longer necessary because the file is part of the framework. Conversely, you'll see new occurrences of `import UITestsFoundation` in the `-Tests.swift` files that use the screen I moved to `UITestsFoundation`.